src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (number) => {
```

Comment:
To properly bind 'this' context in the methods, change arrow functions to regular functions.